public/js/monitor.js

```diff
@@ -36,17 +36,90 @@ function isValidEmail(val) {
 }
 
 
-function doOauth(el) {
+function doOauth(el, {emailWatch = false} = {}) {
+  // Growth Experiment: To sidestep the breach scans form, we have to check if
+  // there is an email address entered into #scan-user-email form.
+  // If so, we set it similiar to what would happen on form submit.
+  //
+  // Options was added to limit how the watched email input field is injected
+  // in this function. It only moves it if  options.emailWatch is set to TRUE;
+
   let url = new URL("/oauth/init", document.body.dataset.serverUrl);
   url = getFxaUtms(url);
-  ["flowId", "flowBeginTime", "entrypoint"].forEach(key => {
+
+  ["flowId", "flowBeginTime", "entrypoint", "form_type"].forEach(key => {
```

Comment:
is `form_type` always `email` ? If so, let's stick this in the oauth controller.

Reply:
Caveat: unless `form_type` needs to also be sent in `View` pings.

Reply:
Revised!